internal/report/report.go

```diff
@@ -1262,6 +1262,9 @@ type Report struct {
 	formatValue func(int64) string
 }
 
+// Profile returns the profile used to create a report.
+func (rpt *Report) Profile() *profile.Profile { return rpt.prof }
```

Comment:
See my comment in webui.go - I don't think this accessor is needed.

Reply:
Done.